### ISSUE TITLE
nushell table output mode

### ISSUE THREAD

**Issue by galop1n** [ogham/exa#1182]
Nushell is a modern take on shell. Nushell's table approach to every thing is neat, and their core `ls` take on the matter is already an improvement or the old way. 

`exa` remains a more advanced `ls` replacement, and it would be terrific to have a nushell compatible output, for example with `exa --nu` that can be pipe the way nushell intends. 

I saw a request for structured exa output that kind of overlap with that request, but felt dissociate enough to open the new issue

**Comment by eggbean** [comment]
Yeah, a `--json` option would be good, like the tree replacement `[tre](https://github.com/dduan/tre)` has.

**Comment by aleksasiriski** [comment]
+1
